### PR TITLE
[Docs] Revamp the documentation, install files, and examples

### PR DESCRIPTION
**Pull request by MikeEdgar**
- Refactor the installation resource files and examples
- Update README with instructions for operator install and example usage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,65 +1,123 @@
...
+
+Modify the `CLUSTER_DOMAIN` to match the base domain of your Kubernetes cluster (used for ingress configuration), use either `route` (OpenShift) or `ingress` (vanilla Kubernetes) for `LISTENER_TYPE`, and set `NAMESPACE` to be the namespace where the Apache Kafka<sup>®</sup> cluster will be created.
+```shell
+export CLUSTER_DOMAIN=apps-crc.testing
+export NAMESPACE=kafka
+export LISTENER_TYPE=route
+cat examples/kafka/*.yaml | envsubst | kubectl apply -n ${NAMESPACE} -f -
+```
+##### Kafka Authorization
+In order to allow the necessary access for the console to function, a minimum level of authorization must be configured for the principal in use for each Kafka cluster connection. While the definition of the permissions may vary depending on the authorization framework in use (e.g. ACLs, Keycloak Authorization, OPA, or custom) the minimum required in terms of ACL types are:
```
Comment (PaulRMellor) [inline README.md:43]:
```suggestion
To ensure the console has the necessary access to function, a minimum level of authorization must be configured for the principal used in each Kafka cluster connection. The specific permissions may vary based on the authorization framework in use, such as ACLs, Keycloak authorization, OPA, or a custom solution. However, the minimum ACL types required are as follows:
```

File: README.md [hunk 2]
```diff
@@ -1,65 +1,123 @@
...
+
+With the operator resources created, you may create a `Console` resource like the one shown in [Console Custom Resource Example](#console-custom-resource-example).
+
+## Running locally
+
+Running the console locally requires configuration of any Apache Kafka<sup>®</sup> clusters that will be accessed from the console and (optionally) the use of a Kubernetes cluster that hosts the Strimzi Kafka operator. To get started, you will need to provide a console configuration file and (optionally) credentials to connect to the Kubernetes cluster where Strimzi is operating.
 
 1. Using the [console-config-example.yaml](./console-config-example.yaml) file as an example, create your own configuration
    in a file `console-config.yaml` in the repository root. The `compose.yaml` file expects this location to be used and
    and difference in name or location requires an adjustment to the compose file.
```
Comment (PaulRMellor) [inline README.md:104]:
```suggestion
   any difference in name or location requires an adjustment to the compose file.
```

**Review by PaulRMellor — APPROVED** [review]
Looks good, @MikeEdgar 
Readme is clear. I left a few suggestions

File: README.md [hunk 3]
```diff
@@ -1,65 +1,123 @@
...
+```shell
+# Enable TLS passthrough on the ingress deployment
+kubectl patch deployment -n ingress-nginx ingress-nginx-controller \
+ --type='json' \
+ -p='[{"op": "add", "path": "/spec/template/spec/containers/0/args/-", "value":"--enable-ssl-passthrough"}]'
+```
+
+### Prerequisites
+#### Kafka
+The instructions below assume an existing Apache Kafka<sup>®</sup> cluster is available to use from the console. We recommend using [Strimzi](https://strimzi.io) to create and manage your Apache Kafka<sup>®</sup> clusters - plus the console provides additional features and insights for Strimzi Apache Kafka<sup>®</sup> clusters.
```
[the remark this answers is not in the document]
Comment (MikeEdgar) [reply README.md:35]: I think so, good catch. I added it to the section header.

**Review by jankalinic — APPROVED** [review]

**Pull request merged**